packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -18,17 +18,18 @@ h| Control
 h| Description
 
 | 1.1 Vulnerability reports
-| * Publish the point of contact for security reports on your website
+| * Publish the method and/or point of contact for security reports on your website
+  ** Example: Using the https://securitytxt.org/)[security.txt] standard
 * Develop and document procedures for triaging reported vulnerabilities
 * Respond to reports within a reasonable time frame
 
 | 1.2 Customer testing
 | * On request, enable your customers or their delegates to test the security of your application
-* Test on a non-production environment if it closely resembles the production environment in functionality
+* Test on a non-production environment that closely resembles the production environment in functionality
 * Ensure non-production environments do not contain production data
 
 | 1.3 Self-assessment
-| Perform annual (at a minimum) security self-assessments using this document
+| Perform annual (at a minimum) security self-assessments using the latest MVSP checklist
```

Comment:
This could be interpreted as the "alpha" version. What about "latest MVSP release" instead?